packages/botonic-plugin-flow-builder/README.md

```diff
@@ -1 +1,104 @@
 # Botonic Plugin Flow Builder
+
+## What Does This Plugin Do?
+
+Botonic Plugin Contentful is one of the **[available](https://github.com/hubtype/botonic/tree/master/packages)** plugins for Botonic.
+Flow Builder is a no-code, drag-and-drop chatbot builder that makes the process of creating and editing chatbot journeys faster and more intuitive.
+From [Hubtype](https://www.hubtype.com/) we offer a suite of tools and features that let you build, test and deploy both automated and agent-based journeys at scale. This plugin allows you to connect to our Flow Builder.  
+
+<p align="center">
+    <img alt="Flow Builder" title="flow-builder" src="https://i.ibb.co/ySRjD4M/Pasted-image-20230426161548.png" width="550">
+</p>
+
+### Features
+
+- Support for a wide variety of content types, including buttons, text, carousels, images, videos, and more.
+- Default function types that can recognize the queue status (open or closed) and the channel type from which the user is writing.
+- The ability to receive custom functions to enable more personalized behaviors.
```

Comment:
I would remove this as will be described above 👆

Reply:
Copy!